.changeset/plenty-pears-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"@solid-primitives/share": major
```

Comment:
you sure you want this to be a major change?

Reply:
Yeah, adding new networks isn't a minor adjustment.

Reply:
Nevermind, I see what you mean.